knative-serving/transport-encryption.adoc

```diff
@@ -0,0 +1,43 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="serving-transport-encryption"]
+= Serving transport encryption
+:context: serving-transport-encryption
+
+toc::[]
+
+{ServerlessProductName} Serving transport encryption feature allows transporting data over secured and encrypted HTTPS connections using TLS.
+
+:FeatureName: {ServerlessProductName} with Python
```

Comment:
"with Python" ?

I don't see how this is related to this document. We have Python in Functions but not in Serving transport encryption.

Reply:
Good catch! Bad copypaste on my part.